zhaquirks/tuya/ts0601_temperaturehumidity.py

```diff
@@ -0,0 +1,114 @@
+"""Tuya temp and humidity sensor with e-ink screen."""
+
+from typing import Dict
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.general import Basic, Groups, Ota, Scenes, Time
+from zigpy.zcl.clusters.measurement import RelativeHumidity, TemperatureMeasurement
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+    SKIP_CONFIGURATION,
+)
+from zhaquirks.tuya import (
+    TuyaLocalCluster,
+    TuyaNewManufCluster,
+    TuyaPowerConfigurationCluster2AAA,
+)
+from zhaquirks.tuya.mcu import DPToAttributeMapping, TuyaDPType
+
+# NOTES:
+# The data comes in as a string on cluster, if there is nothing set up you may see these lines in the logs:
+# Unknown message (b'19830100a40102000400000118') on cluster 61184: unknown endpoint or cluster id: 'No cluster ID 0xef00 on (a4:c1:38:d0:18:8b:64:aa, 1)'
+#                                          28.0 degrees
+# Unknown message (b'19840100a5020200040000022c') on cluster 61184: unknown endpoint or cluster id: 'No cluster ID 0xef00 on (a4:c1:38:d0:18:8b:64:aa, 1)'
+#                                          55.6% humid
+# Unknown message (b'19850100a60402000400000064') on cluster 61184: unknown endpoint or cluster id: 'No cluster ID 0xef00 on (a4:c1:38:d0:18:8b:64:aa, 1)'
+#                                          100% battery
+
+
+class TuyaTemperatureMeasurement(TemperatureMeasurement, TuyaLocalCluster):
+    """Tuya local TemperatureMeasurement cluster."""
+
+
+class TuyaRelativeHumidity(RelativeHumidity, TuyaLocalCluster):
+    """Tuya local RelativeHumidity cluster."""
+
+
+class TemperatureHumidityManufCluster(TuyaNewManufCluster):
```

Comment:
Please extend from `zhaquirks.tuya.mcu.TuyaMCUCluster`.
I have plans to migrate all `TuyaNewManufCluster` to `TuyaMCUCluster` in the long term.

Reply:
done